client/src/app/pages/migration-waves/components/manage-applications-form.tsx

```diff
@@ -202,7 +203,7 @@ export const ManageApplicationsForm: React.FC<ManageApplicationsFormProps> = ({
       applications: selectedItems.map((application) => {
         return { id: application.id, name: application.name };
       }),
-      name: migrationWave?.name.trim() || "",
+      name: migrationWave?.name ? migrationWave.name.trim() : "",
```

Comment:
This is fine as-is, but I think you could have continued the optional chaining for a shorter line. If any part of the optional chain is undefined the whole expression evaluates to undefined, so the `||` is enough to catch you.

```
name: migrationWave?.name?.trim() || "",
```